src/App.jsx

```diff
@@ -66,14 +65,6 @@ function App() {
     setUserIsLogged();
   });
 
-  const signOut = async () => {
-    const clientId = await Config.getGoogleClientId();
-    await GoogleIS.revokeAccessToken(clientId);
-    await Storage.setUserIsLogged(false);
-    await Storage.clearStorage();
-    await setIsLoggedIn(false);
-  };
-
   const signIn = async () => {
```

Comment:
This will move to OIDC with https://broadworkbench.atlassian.net/browse/DCJ-654, correct?

Reply:
Correct. I'm trying to de-tangle the [sign-in code here](https://github.com/DataBiosphere/duos-ui/pull/2611) so it's less of a boil-the-ocean change.